L1Trigger/VertexFinder/python/l1tVertexProducer_cfi.py

```diff
@@ -67,6 +69,10 @@
         VxMinNStub = cms.uint32(4),
         # Minimum number of stubs in PS modules associated to a track
         VxMinNStubPS = cms.uint32(3),
+        # Track weight NN graph 
+        TrackWeightGraph = cms.string(CMSSW_BASE+"/src/L1Trigger/VertexFinder/data/Quantised_model_prune_iteration_9_weightModelgraph.pb"),
+        # Pattern recognition NN graph
+        PatternRecGraph = cms.string(CMSSW_BASE+"/src/L1Trigger/VertexFinder/data/Quantised_model_prune_iteration_9_patternModelgraph.pb"),
```

Comment:
This will need to be adjusted for CMSSW usage and for removing the data files from the local PR. Take a look at: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideEdmFileInPath for the documentation on the recommended mechanism.